python/lsst/meas/deblender/sourceDeblendTask.py

```diff
@@ -460,7 +489,21 @@ def skipParent(self, source, mask):
         """
         fp = source.getFootprint()
         source.set(self.deblendSkippedKey, True)
-        source.set(self.nChildKey, len(fp.getPeaks()))  # It would have this many if we deblended them all
         if self.config.notDeblendedMask:
             mask.addMaskPlane(self.config.notDeblendedMask)
             fp.spans.setMask(mask, mask.getPlaneBitMask(self.config.notDeblendedMask))
+
+        # Set the center of the parent
+        bbox = fp.getBBox()
+        centerX = int(bbox.getMinX()+bbox.getWidth()/2)
+        centerY = int(bbox.getMinY()+bbox.getHeight()/2)
```

Comment:
Are you okay with always rounding down for odd widths/heights, or are they always even?

Reply:
That is a good point, and if this wasn't an approximation anyway then I would definitely want to round to the nearest integer digit (or use a double instead of an int for the column). But for parents that could not be deblended I decided that we should report _some_ position, so the choice was either the centroid or the center of the box. Since the peak locations for all of the properly deblended sources are integer pixel locations, I decided to go for the approximate center of the box, but in practice I don't know that we'll ever actually use the peak_center for these undeblended parents anyway.